src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.ImplicitNamespaceImports.targets

```diff
@@ -0,0 +1,22 @@
+<!--
+***********************************************************************************************
+Microsoft.Android.Sdk.ImplicitNamespaceImports.targets
+
+This file contains @(Import) items for C# 10 global usings.
+
+***********************************************************************************************
+-->
+<Project>
+  <!--
+    There's a master switch, $(DisableImplicitNamespaceImports),
+    which, if set, will prevent all implicit namespace imports.
+    This is done automatically, we don't have to check
+    $(DisableImplicitNamespaceImports) here.
+  -->
+  <ItemGroup Condition=" '$(DisableImplicitNamespaceImports_Android)' != 'true' ">
+    <Import Include="Android.App" />
+    <Import Include="Android.OS" />
```

Comment:
Thus begins the question: *Should* `Android.OS` be in the default namespace list?

It has been in our default template, *but*…. What's been *lacking* in our default templates? `using System`!

Unfortunately, both `System` and `Android.OS` contain an `Environment` type.

I *assume* that if we have both `System` and `Android.OS` "in scope" at the same time, any "unqualified" use of `Environment` will result in a C# error, as it'll be ambiguous.

I also have to assume that `System` is in the default `@(Import)` list.

Thus, the original question: *Should* `Android.OS` be in `@(Import)`?

If we *don't* include it, then we'll need to update our templates to fully qualify `Android.OS.Bundle`.  I think that would be "fine".

Reply:
Yes, I think we should remove `Android.OS` and `Android.Runtime`.

So the list is going to be:
```xml
<Import Include="Android.App" />
<Import Include="Android.Widget" />
```

I thought about `Android.Views` -- but that could cause some upstream complications in dotnet/maui. Our template doesn't even have it *now*, so we probably don't need it.